staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -258,7 +258,7 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 	options := metav1.ListOptions{ResourceVersion: r.relistResourceVersion()}
 
 	if err := func() error {
-		initTrace := trace.New("Reflector ListAndWatch", trace.Field{"name", r.name})
+		initTrace := trace.New("Reflector ListAndWatch", trace.Field{Key: "name", Value: r.name})
```

Comment:
which coding style are you referring to for this change?

Reply:
I guess it's not so much "styling" but it just makes the code more explicit.